languages/common-lisp/exercises/concept/sameness/.meta/example.lisp

```diff
@@ -0,0 +1,34 @@
+(defpackage sameness
+  (:use :cl)
+  (:export
+   :key-object-indentity
+   :key-numbers
+   :key-looser-numbers
+   :key-chars
+   :key-insensitive-chars
+   :key-strings
+   :key-insensitive-string
+   :key-cons-symbols
+   :key-cons-chars
+   :key-cons-numbers
+   :key-cons-insensitive-chars
+   :key-cons-looser-numbers
+   :key-arrays
+   :key-arrays-looser-equal))
+
+(in-package :sameness)
+
+(defun key-object-indentity (x y) (eq x y))
```

Comment:
How do you feel about the functions vs the `case`? I think I'm still generally a `case` fan, but I don't feel too strongly about it.

Reply:
I can really go either way. I think the `case` was good for the 'fill in the blank' style, but the functions are better for this version (which is still *nearly* fill in the blanks - but a bit less.